scripts/generate-build-metadata.bash

```diff
@@ -32,13 +36,17 @@ model=$(jq \
   --arg model "$model_name" \
   --arg release_url "$release_url" \
   --arg release_time "$release_time" \
+  --arg spack_packages_version "$spack_packages_version" \
+  --arg spack_config_version "$spack_config_version" \
   --argjson spack "$spack" \
   '.concrete_specs | to_entries[] | select(.value.name == $model)
   | {
       spack_hash: .key,
       spec: (.value.name + "@" + .value.version),
       created_at: $release_time,
       release_url: $release_url,
+      spack_package: $spack_packages_version,
```

Comment:
```suggestion
      spack_packages: $spack_packages_version,
```
It's a small detail, but we should be consistent with the naming of `spack_packages`